src/v/config/validators.cc

```diff
@@ -67,4 +67,53 @@ validate_connection_rate(const std::vector<ss::sstring>& ips_with_limit) {
     return std::nullopt;
 }
 
+std::optional<std::pair<ss::sstring, int64_t>>
+parse_client_group_byte_rate_quota(const ss::sstring& raw_option) {
```

Comment:
It looks the same for `parse_connection_rate_override`
Maybe reuse it and rename more general?

Reply:
fixed